adobecom/milo
# [MWPW-140627] [MEP] Replace MEP placeholders when loading fragments

When we insert text/HTML in MEP, we call the replacePlaceholders function to replace any already known placeholders (aka MEP placeholders).  We want to do the same thing when it's fragment content that has a MEP placeholder.  So we'll want to call that function again from fragments.js

At the end of fragments.js init function, we are calling handleMepCommands. Afterwards, I think we should test for config.placeholders and fragment's html having a placeholder.  If so, run it through replacePlaceholders.

Note, since there were now 3 if statements that all repeated importing personalization.js for 3 different functions, they were moved to all be nested inside 1 if. 

**QA instructions**
The main element on the test page only uses MEP placeholders, but the gnav also uses regular placeholders. To test this PR, check the performance tab in dev tools for the before and after links. Not only should the LCP be much faster, but it should not call placeholders.json until after LCP in the after link.

Resolves: [MWPW-140627](https://jira.corp.adobe.com/browse/MWPW-140627)

**Test URLs:**
- Before: https://main--cc--adobecom.hlx.page/drafts/mepdev/fragments/2024/q4/fragmentplaceholders/marquee
- After: https://main--cc--adobecom.hlx.page/drafts/mepdev/fragments/2024/q4/fragmentplaceholders/marquee?milolibs=fragmentplaceholders
- Psi: https://fragmentplaceholders--milo--adobecom.hlx.page/?martech=off
